actions/cleanup-branches/README.md

```diff
@@ -0,0 +1,37 @@
+# cleanup-branches
+
+Composite action (step) to get create github app token using vault. This action will query for for branches that are not in an open PR, and will delete them if 'dry-run' is 'false'. Protected branches are excluded as well.
+
+## Inputs
+
+| Name             | Type   | Description                                                                                                                                                                    | Default Value         | Required |
+| ---------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------- | -------- |
+| `token`          | string | GitHub token used to authenticate with `gh`. Requires permission to query for protected branches and delete branches (contents: write) and pull requests (pull_requests: read) | `${{ github.token }}` | true     |
+| `dry-run`        | bool   | If 'dry-run' is not 'true', then the action will print branches to be deleted, but will not delete them                                                                        | `"true"`              | true     |
```

Comment:
```suggestion
| `dry-run`        | `bool`   | If 'dry-run' is `'true'`, then the action will print branches to be deleted, but will not delete them                                                                        | `'true'`              | `true`     |
```

Reply:
composite actions don't really support bools; should it still be listed as a string in this table?